Gemfile

```
@@ -29,6 +29,7 @@ gem "image_processing"
 gem "newrelic_rpm"
 
 gem "omniauth-line_login", path: "omniauth-line_login"
+gem "omniauth-cityos-dcp", git: "https://github.com/TheDesignium/omniauth-cityos-dcp.git", tag: "v1.1.0"
```

```suggestion
gem "omniauth-cityos-dcp", git: "https://github.com/TheDesignium/omniauth-cityos-dcp.git", tag: "v1.1.0"
gem "omniauth-line_login", path: "omniauth-line_login"
```

Gemfile内で並んでいるgemはアルファベット順にするようrubocop側で推奨されているので、この順番にしていただけるとありがたいです！

cf. https://github.com/codeforjapan/decidim-cfj/actions/runs/11548709819/job/32141507445

```
Gemfile:32:1: C: [Correctable] Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem omniauth-cityos-dcp should appear before omniauth-line_login.
gem "omniauth-cityos-dcp", git: "https://github.com/TheDesignium/omniauth-cityos-dcp.git", tag: "v1.1.0"
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

---

ちゃんとrubocop読んでおらず
更新しました